lib/ansible/modules/web_infrastructure/jenkins_script.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+
+# encoding: utf-8
+
+# (c) 2016, James Hogarth <james.hogarth@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+
+DOCUMENTATION = '''
+---
+author: James Hogarth
+module: jenkins_script
+short_description: Executes a groovy script in the jenkins instance
+version_added: '2.2'
+description:
+    - The M(jenkins_script) module takes a script and a hash of values
```

Comment:
grammar cleaned up to make it more readable (yay languages calling dicts hashes and vice versa).